README.md

```diff
@@ -40,13 +40,13 @@ For full usage and API details see the [documentation](https://consensys.github.
 
 ## Examples
 
-The [example](example) directory contains examples of web3js-quorum usage with Besu as a Quorum client.
-The [7nodes-test](7nodes-test) directory contains examples of web3js-quorum usage with GoQuorum as a Quorum client.
+The [example](https://github.com/ConsenSys/web3js-quorum/tree/master/example) directory contains examples of web3js-quorum usage with Besu as a Quorum client.
```

Comment:
we should add a double space at the end of this line otherwise the next line is wrapped to it when it seems to be 2 distinct items.
Or prefix them with `-`, `+`, `*` to make a list.

Reply:
updated, thanks @dolanor for spotting that